python/tvm/contrib/graph_runtime.py

```diff
@@ -69,6 +86,7 @@ class GraphModule(object):
     ctx : TVMContext
```

Comment:
we can safely remove this ctx argument, and ctx from the module, see followup comments

Reply:
in the set_input implementation. change
```
self._set_input(self, value)
```
to
```python
self._get_input(k).copyfrom(value)
```